editor/animation_track_editor.cpp

```diff
@@ -4464,7 +4464,14 @@ AnimationTrackEditor::TrackIndices AnimationTrackEditor::_confirm_insert(InsertD
 
 		} break;
 		case Animation::TYPE_BEZIER: {
-			value = animation->make_default_bezier_key(p_id.value);
+			int existing = animation->track_find_key(p_id.track_idx, time, Animation::FIND_MODE_APPROX);
+			if (existing != -1) {
```

Comment:
```suggestion
			bool existing = animation->track_find_key(p_id.track_idx, time, Animation::FIND_MODE_APPROX) != -1;
			if (!existing) {
```
It can be written like this.

Reply:
That does look cleaner, but the "existing" variable is used in the next line to grab the key value.
`Array arr = animation->track_get_key_value(p_id.track_idx, existing);`

Reply:
Ah okay, so it's fine.